### DISCUSSION TITLE
Message get locked after a moment

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#2023] [Help with using Postal]
Hello,

When I send with several mail servers on Postal, at the beginning it sends with the requested speed but after a while the speed reduces enormously.
I get the following error message **"Couldn't get lock for this message 430339. I won't do this."** with the command **"postal logs worker"** in SSH. This happens several times on several or all workers.

Each time I have to shut down Postal, go to the MySQL database and do the command **"update queued_messages set locked_by = NULL, locked_at = NULL, retry_after = NULL, ip_address_id = NULL, attempts=0;"** and then restart Postal in order to be able to send again normally.

How can I fix this problem please?

**Comment by willpower232** [comment]
Is it possible that you have too many workers for your servers resources? I can imagine this happening if some workers claim a message and then crash before they're able to send the message.
